changelog/fragments/1764777232-ingest-GZIP-logs-is-GA-and-enabled-by-default.yaml

```
@@ -0,0 +1,32 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: feature
+
+# Change summary; a 80ish characters long description of the change.
+summary: "filestream: GZIP support is now GA and enabled by default"
```

Generally in release notes we start with a verb. This is probably what I would do when reviewing the release notes PR so if it makes sense to you, editing it here could eliminate a step later.

```suggestion
summary: "Update GZIP support to GA and enable it by default on filestream"
```